src/main.c

```diff
@@ -2247,7 +2247,7 @@ void on_key(GLFWwindow *window, int key, int scancode, int action, int mods) {
         }
     }
     if (!g->typing) {
-        if (key == CRAFT_KEY_FLY) {
+        if (key == CRAFT_KEY_FLY || key == CRAFT_KEY_HIGH_FLY) {
```

Comment:
I think that the flying behavior should be one key only and that the only thing CRAFT_KEY_HIGH_FLY does is set vy while a character is already flying.

Reply:
This is a one key feature only.